### PR TITLE
S1 task3

### PR DESCRIPTION
**Pull request by JBHarrison5**

### CODE REVIEW

**Review by richardjgabb — APPROVED** [review]
looks good

File: index.php [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+    <a href="index.php" class="logo"><img class="logo" src="images/mcflurryIcon.png" alt="logo"></a>
+    <h1>McFlyWithMe</h1>
+</header>
+<nav class="index">
+    <select name="sort" id="sort">
+        <option value="date">SORT BY DATE</option>
+        <option value="rating">SORT BY RATING</option>
+        <option value="country">SORT BY COUNTRY</option>
+    </select>
+    <form method="get">
```
Comment (matt-ioa) [inline index.php]: `get` is the default method, so no need to specify it. I'm guessing this is a placeholder nav bar in any case...

File: information.php [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+                }
+                $counter -= 1;
+            }
+            for ($i=0; $i < floor(5-$information['rating']); $i++) {
+                echo '<i class="fa-regular fa-star"></i>';
+            }
+        echo '</div>';
+        echo '<p>' . $information['review'] . '</p>';
+        echo '</div>';
+    ?>
```
Comment (matt-ioa) [inline information.php]: You've got some repeated code here, so see if you can turn it into a function, put it into another file and reuse it with `require_once`.

**Review by matt-ioa — CHANGES_REQUESTED** [review]
Just a couple of comments

**Review by matt-ioa — APPROVED** [review]

**Pull request merged**